Add `server_url` parameter for self-hosting Trackio servers (#510)
gradio-app/trackio: docs/source/index.md

  import trackio as wandb
  ```

> - **Local-first** design: dashboard runs locally by default. You can log to a Hugging Face Space (`space_id`) or to a **self-hosted** Trackio server (`server_url` / `TRACKIO_SERVER_URL`). If both are set, the Space wins.

Author: ```suggestion
- **Local-first** design: dashboard runs locally by default. You can also log to a Hugging Face Space (`space_id`) for free or to a **self-hosted** Trackio server (`server_url`)
```
